arm-datalake-store/account/2015-10-01-preview/swagger/account.json

```diff
@@ -757,6 +886,10 @@
           "readOnly": true,
           "description": "the account subscription ID."
         },
+        "identity": {
```

Comment:
Right here is where it is being reference @amarzavery. I just double checked that I didn't spelly it wrong.